docs/_visual/typography.md

```diff
@@ -930,7 +930,8 @@ order: 01
     <h4 class="usa-heading">Implementation</h4>
     <p>To get the max-width on body text, add the class <code>usa-content</code> to your document. Use at the specificity that best suits your project's needs.</p>
     <p>Lists must use <code>usa-content-list</code> for the above.</p>
-    <p>You can change the max-width value <code>$text-max-width</code> in <code>assets/_scss/core/<wbr>variables.scss</code>.</p>
+    <p>You can change the max-width value <code>$text-max-width</code> by
+    overwriting the variables in <code><wbr>_defaults.scss</code>.</p>
```

Comment:
Overwriting should be overriding. 

Need to tell them they should do this in their own custom Sass files and not directly in `_defaults.scss`.